time[] start_times
string[] region_ids
string soma_config
float64[] estimates
string[] contributing_models
